src/TLuaInterpreter.cpp

```diff
@@ -10641,6 +10646,240 @@ int TLuaInterpreter::restartIrc(lua_State* L)
     return 1;
 }
 
+
+/** ttsSpeak( text )
+ *  Synthesizes text for reading.
+ */
+int TLuaInterpreter::ttsSpeak(lua_State* L)
+{
+    TLuaInterpreter::ttsBuild(0);
+    
+    string luaSpeechText = "";
+    if (!lua_isstring(L, 1)) {
+        lua_pushstring(L, "ttsSpeak: wrong argument type");
+        lua_error(L);
+        return 1;
+    } else {
+        luaSpeechText = lua_tostring(L, 1);
+    }
+    
+    QString text(luaSpeechText.c_str());
+    
+    speechUnit->say(text);
+    
+    return 1;
```

Comment:
@keneanung Ah, I was just using them for debugging purposes, returning 1 returns the input if there's no output. Will change 'em back when things are working.